hardhat/tasks/poll-close.ts

```diff
@@ -0,0 +1,39 @@
+import { string } from 'hardhat/internal/core/params/argumentTypes'
+import { task } from 'hardhat/config'
+import { HttpNetworkUserConfig } from 'hardhat/types'
+
+interface CloseArgs {
+  pm: string
+  proposalId: string
+}
+
+/**
+ * Example usage:
```

Comment:
Can you also document the `poll-close` task in README.md and check, if the usage of other tasks is up to date?